spec/models/issue_spec.rb

```diff
@@ -30,6 +30,8 @@
     :url=>"http://localhost///",
     :fragment=>nil}}
 
+  it { should be_kind_of(ErrorStore::Utils) }
```

Comment:
should is not good. And what are you testing here that the issue is an Utils class? I would say to remove this line.